src/routes/admin/wireless/deleteProfile.ts

```diff
@@ -0,0 +1,36 @@
+/*********************************************************************
+ * Copyright (c) Intel Corporation 2019
+ * SPDX-License-Identifier: Apache-2.0
+ * Author : Madhavi Losetty
+ **********************************************************************/
+import Logger from '../../../Logger'
+import { IWirelessProfilesDb } from '../../../repositories/interfaces/IWirelessProfilesDB'
+import { WirelessConfigDbFactory } from '../../../repositories/factories/WirelessConfigDbFactory'
+import { API_RESPONSE, API_UNEXPECTED_EXCEPTION, NETWORK_CONFIG_NOT_FOUND } from '../../../utils/constants'
+import { RPSError } from '../../../utils/RPSError'
+import { EnvReader } from '../../../utils/EnvReader'
+
+export async function deleteWirelessProfile (req, res): Promise<void> {
+  const log = new Logger('deleteWirelessProfile')
+  let wirelessDb: IWirelessProfilesDb = null
+  const { profileName } = req.params
+  wirelessDb = WirelessConfigDbFactory.getConfigDb()
+  try {
+    const results: boolean = await wirelessDb.deleteProfileByName(profileName)
+    if (results) {
+      if (req.secretsManager) {
```

Comment:
secretsManager is required now, do we need this check?

Reply:
Will create a separate PR